README.md

```diff
@@ -54,22 +62,18 @@ The [HiveMind repository](https://github.com/SwarmUS/HiveMind) is included as a
 
 ## Building
 
-Assuming you have a working ROS + Gazebo setup on you system, you need only clone this repository in your Catkin workspace, initialise the git submodules, and build it. 
+Assuming you have a working ROS + Gazebo setup on you system, you need only clone this repository in your Catkin workspace, initialise the git submodules, and build it. You will also need to use [rosdep](http://wiki.ros.org/rosdep) to manage some external package dependencies. Refer to the following instruction for building:
 
 ```
 cd ~/catkin_ws/src
 git clone https://github.com/SwarmUS/SwarmUS-ROS.git
 cd SwarmUS-ROS
 git submodule update --init --recursive
-cd ~/catkin_ws && catkin_cmake
-```
-
-You might need to use [rosdep](http://wiki.ros.org/rosdep) to manage some external package dependencies. The following command installs the dependencies for all the packages of a given workspace :
-
-```
+sh scripts/install dependencies.sh
 rosdep install --from-paths src --ignore-src -r -y
+cd ~/catkin_ws && catkin_cmake
 ```
-
+NOTE: the `install dependencies.sh` script will ask for elevated permissions to install packages.
```

Comment:
Yea and there is some strange stuff going on in the script, like cloning a repo in the usr lib folder. It's a receipe for getting a broken system  (not your fault though, they should provide a better way to install those libs). Put a warning that the user should read the script before usage

Reply:
Yeah, was not a fan of this... I hesitated on forking the dependent repo to change this, but valued my time elsewhere.